webapp/src/pages/Settings.tsx

```diff
@@ -2,58 +2,618 @@ import {
+                  justifyContent="flex-start"
+                  gap={5}
+                >
+                  {displaySubFields("uncertainty", field, value)}
+                </Box>
+              )
+          )}
+      </Box>
+      {divider}
+      {displaySectionTitle("Pipelines")}
+      {resultingConfig?.pipelines &&
+        resultingConfig.pipelines.map(
+          ({ name, model, postprocessors }, index) => (
+            <Box
+              key={index}
+              display="flex"
+              flexDirection="column"
+              gap={1}
+              marginLeft={2}
+              border="1px solid rgba(0, 0, 0, 0.12)"
+            >
+              {displaySectionTitle(name)}
+              <Box
+                display="flex"
+                flexDirection="row"
+                justifyContent="flex-start"
+                gap={5}
+                marginLeft={2}
+              >
+                {displaySubSectionTitle("Model")}
+                {displayReadonlyFields("Class name", model.class_name)}
+                {displayReadonlyFields("Remote", model.remote)}
+              </Box>
+              {model.kwargs && (
+                <Box
+                  display="flex"
+                  flexDirection="row"
+                  justifyContent="flex-start"
+                  gap={5}
+                  marginLeft={2}
+                >
+                  {displaySubSectionTitle("kwargs")}
+                  {Object.entries(model.kwargs).map(([key, value], index) => (
+                    <Box key={index}>
+                      {displayReadonlyFields(key, String(value))}
+                    </Box>
+                  ))}
+                </Box>
+              )}
+              {model.args && model.args.length > 0 && (
+                <Box
+                  display="flex"
+                  flexDirection="row"
+                  justifyContent="flex-start"
+                  gap={5}
+                  marginLeft={2}
+                >
+                  {displaySubSectionTitle("args")}
+                  {Object.entries(model.args).map(([key, value], index) => (
+                    <Box key={index}>
+                      {displayReadonlyFields(key, String(value))}
+                    </Box>
+                  ))}
+                </Box>
+              )}
+              <Box
+                display="flex"
+                flexDirection="column"
+                marginLeft={2}
+                marginBottom={2}
+              >
+                {displayPostprocessorToggleSection({
+                  name,
+                  model,
+                  postprocessors,
+                })}
+                <Box
+                  key={index}
+                  display="flex"
+                  flexDirection="column"
+                  gap={1}
+                  marginLeft={2}
+                >
+                  {postprocessors &&
+                    postprocessors.map((postprocessor, index) => (
```

Comment:
Here the `?` would be best.
```suggestion
                  {postprocessors?.map((postprocessor, index) => (
```